spec/models/public_body_spec.rb

```diff
@@ -532,8 +532,11 @@ def set_default_attributes(public_body)
   before(:each) do
     # Make sure that there's no internal_admin_body before each of
     # these tests:
-    PublicBody.connection.delete("DELETE FROM public_bodies WHERE url_name = 'internal_admin_body'")
-    PublicBody.connection.delete("DELETE FROM public_body_translations WHERE url_name = 'internal_admin_body'")
+    if internal_admin_body = PublicBody::Translation.
```

Comment:
Just destroy all the bodies.

```ruby
PublicBody.destroy_all
```

Check that all translation records are also destroyed.